Changelog

```diff
@@ -1,3 +1,23 @@
+Version 2.22.1, 2018-04-20
+
+  Fixes in WebUI:
+  * Allow to display the messages of several C/R tokens (#995, #1004)
+  * Use ng-if instead of ng-show to avoid errors in the javascript console
+  * Remove reference to not-used system.addons.js to avoid errors in the javascript console
+  * Remove reference to not-used system.addons.html to avoid errors in the javascript console
+  * Avoid request to /false is image is not existing - avoid error in the javascript console
+  * Fix handling of U2F token in the WebUI login
```

Comment:
I guess all the errors above can be linked to #963, plus something like "Use ng-src instead of src to avoid errors in the javascript console"